requirements.yml

```diff
@@ -50,6 +50,9 @@
 - src: git+https://github.com/mother-of-all-self-hosting/ansible-role-focalboard.git
   version: v7.8.0-0
   name: focalboard
+- src: git+https://github.com/mother-of-all-self-hosting/ansible-role-funkwhale.git
+  version: v1.3.0-rc5-3
```

Comment:
We usually avoid "unstable" versions like this, but.. given that:

- this is rc5 and a final release is probably just around the corner
- this is not upgrading any existing users from a stable version to a RC (we don't have any funkwhale users yet)

.. so I suppose it's OK and a good way to proceed.

Reply:
Yes, the idea was that there were some changes in the docker deployment from 1.2 -> 1.3 and 1.3 seemed stable enough to use the rc. Next upgrades will be with the usual stable versions